x-pack/solutions/security/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -546,25 +547,35 @@ export const getEndpointConsoleCommands = ({
       ),
       mustHaveArgs: true,
       args: {
-        action: {
-          required: true,
-          allowMultiples: false,
-          about: i18n.translate(
-            'xpack.securitySolution.endpointConsoleCommands.cancel.action.about',
-            {
-              defaultMessage: 'The response action to cancel',
+        ...(isSupported
```

Comment:
Can you add a tests for this?
Also - make sure you also cover testing of other EDRs to ensure proper handling of this command (should be disabled/"hidden" I think)